_data/glossary/_example.yml

```diff
@@ -1,13 +1,13 @@
 id: _example
 name: Example K8s Term
-formerly:
+aka:
```

Comment:
Say more about using "aka" here. Would it be better to provide an example that matches other glossary entries, and use "related", for example?

Reply:
Hm, could you clarify what you mean by "an example that matches other glossary entries"? Thanks!

Reply:
I suggested aka (also known as). We could add an explanation of the abbreviation in the example file.

Reply:
@chenopis I think we're good as is--I'm probably overthinking it.